### ISSUE TITLE
Keyboard navigation is not scrolling the content of the PanelBar to the active item.

### ISSUE THREAD

**Issue by simonssspirit** [telerik/kendo-ui-core#1884]
### Bug Report
If the PanelBar is scrollable and the keyboard navigation is used, the content does not scroll to the active item.
**Telerik admin ticket**: 1048116

This behavior can be **reproduced** on every demo by making the PanelBar scrollable: 

`#panelbar {
   border: solid 1px;
   height: 400px;
   overflow: scroll;
   overflow-x: auto;
  }`

### Reproduction
1. Use the following [Dojo](https://dojo.telerik.com/@Dimitar-Goshev/ElehIliX)
1. Click on the first PanelBar node.
1. Press down arrow to move the focus to not visible items

### Current behavior
Not visible nodes are not scrolled into view, when focused.

### Desired behavior
Not visible nodes are scrolled into view, when focused.

### Environment
**Kendo UI:** 2023.3.1114
**Browser:** all
